AEPServices/Sources/ui/UIServiceInterface.swift

```diff
@@ -0,0 +1,43 @@
+/*
+ Copyright 2020 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+
+/// Interface for displaying alerts, local notifications, and fullscreen web views
+public protocol UIServiceInterface {
```

Comment:
Although protocols are technically interfaces, it's best practice in Swift to not name protocols interface. I would just remove the `interface` from all the protocols you've created. I know it makes naming the implementations tricker but just look at how we name our other services.